src/Block.php

```diff
@@ -34,26 +64,46 @@ public function channelLinks() {
     foreach ($enabled_channels as $channel_name) {
       $channel_options = isset($options[$channel_name]) ? $options[$channel_name] : array();
       $channel = $loader->channel($channel_name, $this, $channel_options);
-      if($channel->enabled() && ($channel_link = $channel->render())) {
+      if ($channel->enabled() && ($channel_link = $channel->render())) {
         $links[$channel_name] = $channel_link;
       }
     }
     return $links;
   }
 
+  /**
+   * Returns the absolute URL pointing to the containing node.
+   */
   public function getUrl($absolute = TRUE) {
     $link = $this->options['link'];
     return url($link['path'], $link + array('absolute' => $absolute));
   }
 
+  /**
+   * Returns the relative URL pointing to the containing node.
+   *
+   * @return array
+   *   `path`: String of format 'node/[node:nid]'
+   */
   public function getLink() {
     return $this->options['link'];
   }
 
+  /**
+   * Returns the containing node's object.
+   */
   public function getNode() {
     return $this->options['node'];
   }
 
+  /**
+   * Returns the URL pointing to the block's image.
```

Comment:
I think it rather should be "the share image". It’s part of the share-content. It’s not displayed as part of the block.